invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.overrides

```diff
@@ -68,6 +68,10 @@ button:focus-visible, a:focus-visible {
   box-shadow: 0 2px 5px rgba(0, 0, 0, 0.2);
 }
 
+.wrap-overflowing-text {
+  word-wrap: break-word;
```

Comment:
how will word wrapping look like with multiple lines? will it break any layout?

Reply:
after this change it can. What will happen if you add even more lines? how does it look like in the request details view ? in the searches ?

Reply:
i will test it :+1:

Reply:
So `adding lines` does not really change anything it just a long string. This only affects the record details page.